workspace/assets/js/utils/global.js

```diff
@@ -49,7 +49,11 @@
 			'android',
 			'phone',
 			'tablet',
-			'touch'
+			'chrome',
+			'safari',
+			'internetexplorer',
+			'firefox',
+			'edge'
```

Comment:
This is:

1. Not needed: just add that the keys file into the pruner.
2. Unrelated to this PR ?

Reply:
> and added things to our cheap modrnzr
Yeah it's related, I should have called the PR _unbreak all the things the Framework v2 broke_